### PR TITLE
[MST-494] Clarify photo ID instructions with example image

### PR DESCRIPTION
**Pull request by bseverino**
[MST-494](https://openedx.atlassian.net/browse/MST-494)

This adds an example image of a photo ID card and updates the copy in order to clarify what photo ID means. I've also removed an outdated comment on the submitted panel.

---
Screen Shot 2020-12-16 at 1 09 12 PM

---
Screen Shot 2020-12-16 at 1 43 07 PM

### CODE REVIEW

File: src/id-verification/IdVerification.messages.js [hunk 1]
```diff
@@ -33,7 +38,7 @@ const messages = defineMessages({
   },
   'id.verification.requirements.card.id.text': {
     id: 'id.verification.requirements.card.id.text',
-    defaultMessage: 'You need a valid ID that contains your full name and photo.',
+    defaultMessage: 'You need a valid identification card that contains your full name and photo.',
```
Comment (MichaelRoytman) [inline src/id-verification/IdVerification.messages.js:41]: I could be wrong here, but I vaguely recall that if a particular message has been translated for a given key, then modifying the `defaultMessage`  for that key doesn't result in the message being re-translated. I recall needing to specify a new key. This is my memory from an issue from three years ago, though, so I could be off base. Do you know anything about that?

Comment (bseverino) [reply src/id-verification/IdVerification.messages.js:41]: Not sure but it sounds likely - I'll see if there's anything in the docs

Comment (bseverino) [reply src/id-verification/IdVerification.messages.js:41]: Okay, based on the info here it should flag it to be translated again if the value changes but the key stays the same: https://docs.transifex.com/projects/updating-content

Comment (MichaelRoytman) [reply src/id-verification/IdVerification.messages.js:41]: Great. Thank you for checking!

**Review by MichaelRoytman — APPROVED** [review]

**Pull request merged**